plugins/src/test/kotlin/org/gradle/kotlin/dsl/plugins/AbstractPluginTest.kt

```diff
@@ -15,11 +15,11 @@ open class AbstractPluginTest : AbstractIntegrationTest() {
     fun setUpTestPluginRepository() {
         val testRepository = normaliseFileSeparators(absolutePathOf("build/repository"))
         val futureVersion = loadTestProperties()["version"]
-        withFile("settings.gradle", """
+        withFile("settings.gradle.kts", """
             pluginManagement {
                 repositories {
-                    maven { url = uri("$testRepository") }
-                    maven { url = uri("https://jcenter.bintray.com/") }
+                    maven { setUrl("$testRepository") }
```

Comment:
:o: Let's not forget to add follow up issues to recover the nice DSL.

Reply:
https://github.com/gradle/kotlin-dsl/issues/543